charts/ztka/templates/configmap-fluentbit.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentbit-config
+data:
+  custom_parsers.conf: |
+    [PARSER]
+        Name docker_no_time
+        Format json
+        Time_Keep Off
+        Time_Key time
+        Time_Format %Y-%m-%dT%H:%M:%S.%L
+  fluent-bit.conf: |
+    [SERVICE]
+        Daemon Off
+        Flush 1
+        Log_Level info
+        Parsers_File parsers.conf
+        Parsers_File custom_parsers.conf
+        HTTP_Server On
+        HTTP_Listen 0.0.0.0
+        HTTP_Port 2020
+        Health_Check On
+    [INPUT]
+        Name tail
+        Tag system
+        Path /var/log/containers/*paralus*paralus-tail*.log
+        Refresh_Interval 5
+        DB /var/log/fluentbit.db
+        DB.Sync Normal
+        Mem_Buf_Limit 5M
+        Skip_Long_Lines On
+        Read_from_Head True
+        Parser json
+    [INPUT]
+        Name tail
+        Tag kubectl_api
+        Path /var/log/containers/*relay-server*relay-tail*.log
+        Refresh_Interval 5
+        DB /var/log/fluentbit.db
+        DB.Sync Normal
+        Mem_Buf_Limit 5M
+        Skip_Long_Lines On
+        Read_from_Head True
+        Parser json
+    [INPUT]
+        Name tail
+        Tag kubectl_cmd
+        Path /var/log/containers/*prompt*prompt-tail*.log
+        Refresh_Interval 5
+        DB /var/log/fluentbit.db
+        DB.Sync Normal
+        Mem_Buf_Limit 5M
+        Skip_Long_Lines On
+        Read_from_Head True
+        Parser json
+    [FILTER]
+        Name kubernetes
+        Match kube.*
+        Kube_URL https://kubernetes.default.svc:443
+        Kube_CA_File /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        Kube_Token_File /var/run/secrets/kubernetes.io/serviceaccount/token
+        Merge_Log Off
+        K8S-Logging.Parser On
+        K8S-Logging.Exclude On
+        Annotations off
+        Labels off
+    [FILTER]
+        Name record_modifier
+        Match *
+        Remove_key kubernetes*
+        Remove_key stdout
+
+    [OUTPUT]
+        Name stdout
+        Match kube.*
+        Format json
+        Json_Date_Key timestamp
+        Json_Date_Format iso8601 
+
+    [OUTPUT]
+        Name pgsql
+        Match *
+        Host {{ include "ztka.dbAddr" . | }}
+        Port 5432
+        User {{ include "ztka.dbUser" . | }}
+        Password {{ include "ztka.dbPassword" . | }}
+        Database {{ include "ztka.dbName" . | }}
+        Table paralus
```

Comment:
Use some different table name, like `audit_logs`?

Reply:
yes, i was thinking the same will change this in the next commit